s3/replication/manager/src/s3replicationmanager/job_routes.py

```diff
@@ -120,15 +120,36 @@ async def get_jobs(request):
 
     all_jobs_list = request.app['all_jobs']
 
-    if 'queued' in query:
+    if 'count' in query:
+        if 'queued' in query['count']:
```

Comment:
This should be `if 'queued' in query:` right?

Reply:
as I was passing 'count=queued' in the request, query['count'] will return 'queue'.

Reply:
fixed.